code/modules/dataloader/data_loader.py

```diff
@@ -434,13 +438,17 @@ def get_chunks(self, uploaded_files, weblinks):
     ]
 
     data_loader = DataLoader(config, logger=logger)
-    document_chunks, document_names, documents, document_metadata = (
-        data_loader.get_chunks(
-            [
-                "https://dl4ds.github.io/fa2024/static_files/discussion_slides/00_discussion.pdf"
-            ],
-            [],
-        )
+    # Just for testing
+    (
+        document_chunks,
+        document_names,
+        documents,
+        document_metadata,
+    ) = data_loader.get_chunks(
+        [
+            "https://dl4ds.github.io/fa2024/static_files/discussion_slides/00_discussion.pdf"
```

Comment:
Still hard coded path?

Reply:
That is just to test the module, it is not used in the pipeline

Reply:
I can add that as an arg

Reply:
Fixed @trgardos

Reply:
Oops! Format issue.